### PR TITLE
Fetching Quote from API instead of locally, and small Stylistic Changes

### LINKED ISSUE #2: Add many more quotes
Maybe import some from [Wikiquote:Quote of the Day](https://en.wikiquote.org/wiki/Wikiquote:Quote_of_the_Day)

### PR DESCRIPTION
**Pull request by Jawkx**
I saw that that you need to add more quote towards the respo manually, and I think instead of that why not just grab it from an API. The API I used are [Quotable](https://github.com/lukePeavey/quotable).

The `QuoteMachine.tsx` was redundant now because the API have more than 9000 quotes.

The `generateQuote` method is changed from using `QuoteMachine` to a simple fetch function from the API.

``` javascript
  generateQuote(): void {
    fetch('https://api.quotable.io/random')
      .then(response => response.json())
      .then(data => {
        this.setState({
          quote: {
            Text: data.content,
            Author: data.author
          }
        })
      })
    this.setState({
      quote: {
        Text: "Fetching data...",
        Author: "Fetching data..."
      }
    })
  }
```

Basically when `generateQuote` is used, it will fetch a data from `'https://api.quotable.io/random'`

I also added abit of stylistic change at the footer components. Center the text and make the font smaller.

### CODE REVIEW

**Review by janreggie — CHANGES_REQUESTED** [review]
This actually looks good, and I appreciate that you took the time to writing this pull request. My initial goal for randomquoter was just simply "take some quotes and make sure the tests work". I wanted to expand my pool of quotes but never really had the time. My initial goal was to parse some from Wikiquote and serve them statically, although taking them from an API does seem more sensible.

I would request that you change the notice in the Footer, since now we're sourcing them from quotable. Similarly, a link to their website is helpful.

I am wondering though, if a quotable-based `QuoteMachine` could be made so that quotes can just be `generate()`'d, which can be done by turning `QuoteMachine` into an interface, but perhaps that's a change to be done in the future.

**Comment by Jawkx** [comment]
I will make change to the footer, but can you merge it first? Hahaha, I need pull request for Hacktoberfest lol

**Comment by janreggie** [comment]
Oh, is that a thing? Well then...
This PR resolves #2.
